src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -316,6 +316,43 @@ internal static SemanticVersion PSV6Version
     }
 
     /// <summary>
+    /// Represents an implementation of '$PSVersionTable' variable.
+    /// The implementation contains ordered 'Keys' and 'GetEnumerator' to get user-friendly output.
+    /// </summary>
+    public sealed class PSVersionHashTable : Hashtable, IEnumerable
+    {
+        internal PSVersionHashTable(IEqualityComparer equalityComparer) : base(equalityComparer)
+        {
+        }
+
+        /// <summary>
+        /// Returns ordered collection with Keys of 'PSVersionHashTable'
+        /// </summary>
+        public override ICollection Keys
+        {
+            get
+            {
+                Array arr = new string[base.Keys.Count];
+                base.Keys.CopyTo(arr, 0);
+                Array.Sort(arr, StringComparer.OrdinalIgnoreCase);
```

Comment:
I'm not sure I like this sorting - today `PSVersion` shows up first, or at least it should.
With this change, it will be somewhat harder to see